context/app/static/js/pages/Dataset/Dataset.tsx

```diff
@@ -125,36 +129,39 @@ function DatasetDetail({ assayMetadata }: EntityDetailProps<Dataset>) {
   return (
     <DetailContextProvider hubmap_id={hubmap_id} uuid={uuid} mapped_data_access_level={mapped_data_access_level}>
       <SelectedVersionStoreProvider initialVersionUUIDs={processedDatasets?.map((ds) => ds._id) ?? []}>
-        <ExternalDatasetAlert isExternal={Boolean(mapped_external_group_name)} />
-        {Boolean(is_component) && <ComponentAlert />}
-        <DetailLayout sections={shouldDisplaySection} isLoading={isLoading}>
-          <Summary
-            entityTypeDisplay="Dataset"
-            status={combinedStatus}
-            mapped_data_access_level={mapped_data_access_level}
-            mapped_external_group_name={mapped_external_group_name}
-            bottomFold={
-              <>
-                <MultiAssayRelationship assay_modality={assay_modality} />
-                {shouldDisplayRelationships && (
-                  <Box height={400} width="100%" component={Paper} p={2}>
-                    <DatasetRelationships uuid={uuid} processing={processing} />
-                  </Box>
-                )}
-              </>
-            }
-          >
-            <SummaryDataChildren mapped_data_types={mapped_data_types} mapped_organ={mapped_organ} />
-          </Summary>
-          <MetadataSection shouldDisplay={shouldDisplaySection.metadata} />
-          <ProcessedDataSection shouldDisplay={Boolean(shouldDisplaySection['processed-data'])} />
-          <BulkDataTransfer shouldDisplay={Boolean(shouldDisplaySection['bulk-data-transfer'])} />
-          <ProvSection shouldDisplay={shouldDisplaySection.provenance} />
-          <CollectionsSection shouldDisplay={shouldDisplaySection.collections} />
-          <Attribution>
-            <ContributorsTable contributors={contributors} contacts={contacts} />
-          </Attribution>
-        </DetailLayout>
+        <SelectableTableProvider tableLabel="Current Dataset">
```

Comment:
Does the table provider need to wrap everything or just the `AddDatasetsFromSearchOrDetailDialog`?